### PR TITLE
Fixes #1629 : Developed eventyay iconary, fixed broken icon & dimensions

### LINKED ISSUE #1629: Fix the broken eventyay icon
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

Broken **eventyay** icon in menu. (Also, in wrong dimensions. Should be in `48x48`)

**Expected behavior:**
Should display and can navigate to eventyay

---
**Screenshots of the issue:**

image

### PR DESCRIPTION
**Pull request by 0x48piraj**
# Fixes #1629 

### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

### Changes proposed in this pull request:

image

- Procured the **eventyay** logo, fixed dimensions, enhanced quality.
- Fixed the icon, adjusted it to be responsive :tada:

### CODE REVIEW

File: src/www/index.html [hunk 1]
```diff
@@ -183,7 +183,7 @@
         </div>
         <hr style="margin-bottom: 0; margin-top: 10px;">
         <div class="custom-menu-item">
-          <a class="custom-icon" href="https://eventyay.com/" target="_blank"><img src="./icons/eventyay.png">
+          <a class="custom-icon" href="https://eventyay.com/" target="_blank"><img src="./favicon.ico">
```
Comment (geekyd) [inline src/www/index.html]: @0x48piraj Please use eventyay.png only, instead of changing logo resize it to proper dimensions.

Comment (0x48piraj) [reply src/www/index.html]: @geekyd It's outdated icon.
It been adapted in many projects.

**Comment by 0x48piraj** [comment]
Ping @shubham-padia

**Comment by geekyd** [comment]
@0x48piraj Please check the logo for eventyay https://eventyay.com.

**Comment by 0x48piraj** [comment]
image


help me, I'm unable to see !

**Comment by geekyd** [comment]
@0x48piraj 

screen shot 2018-01-14 at 12 23 09 am

**Comment by 0x48piraj** [comment]
@geekyd Should I close this ?

**Comment by geekyd** [comment]
@0x48piraj Yes for now we can close this.

**Comment by 0x48piraj** [comment]
Thanks ! :100:

**Pull request closed without merging**